### ISSUE TITLE
BUG/ENHANCEMENT: Drop first part at 0,0,0 option

### ISSUE THREAD

**Issue by nathaneltitane** [leozide/leocad#548]
Dropping a part into the editor window or adding subassemblies to already existing models often results in them dropping at astronomical distances from the target.

See here the example of a new submodel, for which part 1, at step one is dropped god knows where into the main ...

An option or feature to have it drop at 0,0,0 or to a close/restrained orbit around the origin would be beneficial.

Thanks 

![Screenshot_20201211-215456_VNC Viewer](https://user-images.githubusercontent.com/5674757/101970977-cdc4f800-3bfb-11eb-8b3e-5e9aec4deb14.jpg)

**Comment by nathaneltitane** [comment]
@leozide thoughts on this? part inserts are really wonky due to this same occurrence - they land super far and when resetting views, models become undistinguishable - undo is the only option

**Comment by leozide** [comment]
This always works for me, parts are placed in a reasonable location.

**Comment by nathaneltitane** [comment]
We had discussed this a while back and frankly, this occurs way to often for me for some reason. You can look at the coordinates on screen cap to tell you.

**Comment by nathaneltitane** [comment]
@leozide would it have anything to do with the mouse sensitivity setting?

**Comment by nathaneltitane** [comment]
@leozide 
toggled the mouse sensitivity to its lowest setting and so far so good.

What was your setting on just for reference?

**Comment by leozide** [comment]
Sensitivity doesn't matter, it's related to the camera angle and where the mouse intersects the base plane.

**Comment by nathaneltitane** [comment]
Right, so when generating a new submodel the orientation is set to the default angle...

I agree the preview doesn't reflect that but that's usually when it'll happen, not to mention when positioning other pieces while editing

**Comment by rsbx** [comment]
Nathan,

What projection mode are your using when this happens?

I've had it happen to me but I noticed that the piece was being put on 
the base plane where the cursor was, as Leo said earlier, WHEN I dragged 
the piece from the parts list. Front, Back, Left, and Right views were 
particularly problematic as was using the orthogonal projection mode.

My work around is the highlight the part I want and hit the 'insert' 
key. The part is then dropped on top of whatever is at 0,0,*. Or to 
shift the view to be more above than to the side of the model.
